frontend/src/types/CampsTypes.ts

```diff
@@ -0,0 +1,40 @@
+export type Camp = {
+  id: string;
+  active: boolean;
+  ageLower: number;
+  ageUpper: number;
+  campCoordinators: string[];
+  campCounsellors: string[];
+  name: string;
+  description: string;
+  earlyDropoff: string;
```

Comment:
Is it ok for these fields to be strings? In the future, we'll need to create 30 minute timeslots from Early Dropoff time to start time AND endTime to latePickup time. Would storing this as DateTime (with a constant Date) make things easier in the future? CC: @annechung @AmandaMisjuwar

Reply:
Yea if we need to do date/time calculations we should store them as Dates (however in Mongo they're currently also strings). But if they're always 30 min time slots we could also just convert from string -> Date -> string whenever we do the calculation? @annechung @dhmi19

Reply:
I feel like it makes more sense to store as strings since these fields are actually times that aren’t tied to a specific date?

Reply:
Yeah we'll just have to make sure theyre in a format that can be converted into Date if needed for calculation (will leave it as string for now)